manual/src/cmds/top.etex

```diff
@@ -26,10 +27,10 @@ directive-argument:
         | 'true' || 'false'
 \end{syntax}
 
-A phrase can consist of a definition, like those found in
+A phrase can consist of a sequence of definitions, like those found in
```

Comment:
Do I correctly understand that this change is correct but unrelated to the present PR? (The PR implements "several phrases per prompt", and here you add an independent remark that `let x = 1 let y = 2` counts as a single phrase?)

Reply:
This is correct. Same for the change to the grammar fragment, where the sequence of definitions can be empty.